src/renderer/utils/constants.ts

```diff
@@ -20,9 +20,16 @@ export type VIZUALISATION =
 export const ACCEPTED_EXTENSION = ".pst";
 export const ARBITRARY_FLAT_LEVEL = 12;
 
+export const ROOT = "root";
+
 // colors
 export const RED = "red";
 export const TRANSPARENT = "transparent";
+export const KEEP_COLOR = "#62bc6f";
```

Comment:
Maybe separate constants by usage ; either with a prefix or a namespace (or as const);  
We should maybe also considering separating them into multiple files.